src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+let rowCount = table.rows.length;
+let colCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row and one column initially. Accessing `table.rows[0].cells.length` will throw an error if there are no rows in the table.